vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/payment/post_order/post_order_unittest.cc

```diff
@@ -79,15 +79,15 @@ TEST_F(PostOrderTest, ServerOK) {
             callback(response);
           }));
 
-  type::SKUOrderItem item;
-  item.quantity = 4;
-  item.sku = "asdfasfasfdsdf";
-  item.type = type::SKUOrderItemType::SINGLE_USE;
-  std::vector<type::SKUOrderItem> items;
-  items.push_back(item);
+  type::SKUOrderItemPtr item = ledger::type::SKUOrderItem::New();
```

Comment:
I don't know where this ledger::type prefix is coming from, but we should also be using ledger::mojom::SKUOrderItemPtr

Reply:
`ledger::type` was being created in `vendor/bat-native-ledger/include/bat/ledger/mojom_structs.h` - It looks like this is DEPRECATED, I'll update the test to use `mojom` directly